algo.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo -e "\033[0m"
+    exit 1
+}
+
+
+if [ $# -eq 0 ]; then
+    cat ./script/help.txt
+fi
+
+command=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+
+if [ "$command" = "help" ]; then
+    cat ./script/help.txt
```

Comment:
If you really want to make this robust, consider setting a variable to the script running directory, e.g., 

SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"

Then, you would do things like "cat ${SCRIPT_DIR}/script/help.txt". The reason is that it may not always be that the script is run in the same directory that it is in, e.g., "./algo.sh". If the script is run from a different directory, the relative paths would break. You may then want to propagate this logic to the other bash scripts that rely on relative paths.